Library/Homebrew/cmd/pull.rb

```diff
@@ -126,42 +120,58 @@ def pull
+        unless message.include? "Closes ##{issue}."
+          message += "\nCloses ##{issue}."
+        end
+      end
 
-        if ARGV.include? "--bump"
-          odie "Can only bump one changed formula" unless changed_formulae.length == 1
-          formula = changed_formulae.first
-          subject = "#{formula.name} #{formula.version}"
+      if changed_formulae.empty?
+        odie "cannot bump: no changed formulae found after applying patch" if do_bump
+        is_bumpable = false
+      end
+      if is_bumpable && !ARGV.include?("--clean")
+        formula = changed_formulae.first
+        new_versions = {
+            stable: formula.stable.nil? ? nil : formula.stable.version.to_s,
+            devel: formula.devel.nil? ? nil : formula.devel.version.to_s
+        }
+        orig_subject = message.empty? ? "" : message.lines.first.chomp
+        subject = subject_for_bump(formula, old_versions, new_versions)
+        if do_bump
+          odie "No version changes found for #{formula.name}" if subject.nil?
           ohai "New bump commit subject: #{subject}"
-          system "/bin/echo -n #{subject} | pbcopy"
+          system "/bin/echo -n '#{subject}' | pbcopy"
           message = "#{subject}\n\n#{message}"
+        elsif subject != orig_subject && !subject.nil?
+          opoo "Nonstandard bump subject: #{orig_subject}"
+          opoo "Subject should be: #{subject}"
         end
+      end
 
-        # If this is a pull request, append a close message.
-        unless message.include? "Closes ##{issue}."
-          message += "\nCloses ##{issue}."
-          safe_system "git", "commit", "--amend", "--signoff", "--allow-empty", "-q", "-m", message
-        end
+      if message != orig_message && !ARGV.include?("--clean")
+        safe_system "git", "commit", "--amend", "--signoff", "--allow-empty", "-q", "-m", message
       end
 
       if fetch_bottles
         bottle_commit_url = if testing_job
-          bottle_branch = "testing-bottle-#{testing_job}"
-          url
-        else
-          bottle_branch = "pull-bottle-#{issue}"
-          if tap.core_formula_repository?
-            "https://github.com/BrewTestBot/homebrew/compare/homebrew:master...pr-#{issue}"
-          else
-            "https://github.com/BrewTestBot/homebrew-#{tap.repo}/compare/homebrew:master...pr-#{issue}"
-          end
-        end
+                              bottle_branch = "testing-bottle-#{testing_job}"
+                              url
+                            else
+                              bottle_branch = "pull-bottle-#{issue}"
+                              if tap.core_formula_repository?
+                                "https://github.com/BrewTestBot/homebrew/compare/homebrew:master...pr-#{issue}"
+                              else
+                                "https://github.com/BrewTestBot/homebrew-#{tap.repo}/compare/homebrew:master...pr-#{issue}"
+                              end
+                            end
```

Comment:
Changed. Pulled the `if`on to the next line to allow aligning with the control structure but keeping everything off to the left and close to the variable.